src/lib/swapper/swappers/LifiSwapper/utils/getNetworkFeeCryptoBaseUnit/getNetworkFeeCryptoBaseUnit.ts

```diff
@@ -88,7 +88,9 @@ export const getNetworkFeeCryptoBaseUnit = async ({
   const networkFeeCryptoBaseUnit = calcNetworkFeeCryptoBaseUnit({
     ...average,
     supportsEIP1559,
-    gasLimit: gasLimit?.toString(),
+    // Conform to Li.Fi SDK's gasLimit buffer
+    // https://github.com/lifinance/sdk/blob/ca2c5d6fa789346d415f1d55a4b9bde15c683385/src/allowance/utils.ts#L60
+    gasLimit: bnOrZero(gasLimit?.toString()).times(1.25).toString(),
```

Comment:
This is wrong and should be for approvals only - needs to be discriminated on the `lifiStep`

Reply:
Actually remove me altogether - this util only handles akschual LiFi steps, not approvals